etc/omero.properties

```diff
@@ -204,7 +204,7 @@ omero.policy.binary_access=+read,+write,+image
 ## set the url to an empty string.
 ##
 ## For more information, see
-## http://www.openmicroscopy.org/site/support/omero/sysadmins/UpgradeCheck.html
+## https://docs.openmicroscopy.org/latest/omero/omero/sysadmins/UpgradeCheck.html
```

Comment:
I think the right one should be

https://docs.openmicroscopy.org/latest/omero/sysadmins/UpgradeCheck.html

-> remove one ``/omero`` from your url